### DISCUSSION TITLE
Show social providers buttons on register page using the new User Profile

### DISCUSSION THREAD

**Discussion by derlin** [keycloak/keycloak#22767] [Q&A]
With the old declarative user profile, it was possible to customize the `register.ftl` to show identity providers buttons, through the use of `social.providers`. However, this information is not available when using the new "User Profile" feature (in preview). That is, adding the same to the `register-user-profile.ftl` doesn't show any button.

This is really problematic, as it hinders the user experience. Someone lends on "Sign In" and sees Google, but when clicking "Register" the option disappears.

Do you have any idea on how to get the social buttons back using the new User Profile?

**Comment by aankansah** [comment]
> With the old declarative user profile, it was possible to customize the `register.ftl` to show identity providers buttons, through the use of `social.providers`. However, this information is not available when using the new "User Profile" feature (in preview). That is, adding the same to the `register-user-profile.ftl` doesn't show any button.
> 
> This is really problematic, as it hinders the user experience. Someone lends on "Sign In" and sees Google, but when clicking "Register" the option disappears.
> 
> Do you have any idea on how to get the social buttons back using the new User Profile?

I have the same issue. Please any help on this?

**Comment by tommygiesbrecht** [comment]
Any news on this?

**Comment by Shreyaan-autonomix** [comment]
Any news on this?
